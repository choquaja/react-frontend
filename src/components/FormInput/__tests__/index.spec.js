import React from 'react';
import { shallow } from 'enzyme';
import FormInput from '../';

describe('FormInput', () => {
  it('renders without crashing', () => {
    shallow(<FormInput />);
  });

  it('render with text', () => {
    const withtext = shallow(<FormInput type="hello" />);
    expect(withtext).toMatchSnapshot();
  });

  it('render with block true', () => {
    const withtext = shallow(<FormInput block />);
    expect(withtext).toMatchSnapshot();
  });

  it('render with full true', () => {
    const withtext = shallow(<FormInput full />);
    expect(withtext).toMatchSnapshot();
  });

  it('render with large true', () => {
    const withtext = shallow(<FormInput large />);
    expect(withtext).toMatchSnapshot();
  });

  it('render with small true', () => {
    const withtext = shallow(<FormInput small />);
    expect(withtext).toMatchSnapshot();
  });

  it('render with grouped true', () => {
    const withtext = shallow(<FormInput grouped />);
    expect(withtext).toMatchSnapshot();
  });
});
