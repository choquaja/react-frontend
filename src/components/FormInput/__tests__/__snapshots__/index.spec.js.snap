// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormInput render with block true 1`] = `
<input
  className="FormInput__FormInput-hffHYv brUyRq"
  type="text"
/>
`;

exports[`FormInput render with full true 1`] = `
<input
  className="FormInput__FormInput-hffHYv jiuqWu"
  type="text"
/>
`;

exports[`FormInput render with grouped true 1`] = `
<input
  className="FormInput__FormInput-hffHYv jLxrLJ"
  type="text"
/>
`;

exports[`FormInput render with large true 1`] = `
<input
  className="FormInput__FormInput-hffHYv kumkId"
  type="text"
/>
`;

exports[`FormInput render with small true 1`] = `
<input
  className="FormInput__FormInput-hffHYv cordqx"
  type="text"
/>
`;

exports[`FormInput render with text 1`] = `
<input
  className="FormInput__FormInput-hffHYv ibYDkF"
  type="hello"
/>
`;
